apps/templates/content_templates.py

```diff
@@ -294,9 +311,9 @@ def render_content_template(item, template):
         elif not (isinstance(value, dict) or isinstance(value, list)):
             updates[key] = value
 
-    if template.get('template_desk'):
+    if template.get('template_desks') and len(template['template_desks']) == 1:
         updates['task'] = {}
-        updates['task']['desk'] = template.get('template_desk')
+        updates['task']['desk'] = template['template_desks'][0]
```

Comment:
Hi @mugurrus, does this line mean that when a story is created from a template, it will always be created in the first desk of the list? If so what happens:
- If the selected desk at the top dropdown is different?
- If user is not a member of the first desk?